CHANGELOG.rst

```diff
@@ -1,5 +1,10 @@
 CHANGELOG for molecule
 ======================
+1.0.7.dev1
+----
```

Comment:
Do we really need a changelog entry for this?  Seems this work can go out as 1.1.0 once  the 'static' provider is done, no?  It's all prep work for that.